core/controlplane/config/templates/cloud-config-controller

```diff
@@ -2578,6 +2654,89 @@ write_files:
                     - --container=heapster
                     - --poll-period=300000
 
+  - path: /srv/kubernetes/manifests/metrics-server-sa.yaml
+    content: |
+        apiVersion: v1
+        kind: ServiceAccount
+        metadata:
+          name: metrics-server
+          namespace: kube-system
+          labels:
+
+  - path: /srv/kubernetes/manifests/metrics-server-de.yaml
+    content: |
+        apiVersion: extensions/v1beta1
+        kind: Deployment
+        metadata:
+          name: metrics-server
+          namespace: kube-system
+          labels:
+            k8s-app: metrics-server
+          annotations:
+            scheduler.alpha.kubernetes.io/critical-pod: ''
+        spec:
+          selector:
+            matchLabels:
+              k8s-app: metrics-server
+          template:
+            metadata:
+              name: metrics-server
+              labels:
+                k8s-app: metrics-server
+            spec:
+              serviceAccountName: metrics-server
+              containers:
+              - name: metrics-server
+                image: gcr.io/google_containers/metrics-server-amd64:v0.2.0
```

Comment:
shouldn't image be templated from  a value from cluster.yaml?

Reply:
yes, fixed in #1035